news/2024/2024-06-17-the-followpoint-dressurf-top-keyboard-smasher

```diff
@@ -0,0 +1,110 @@
+>
+> Upon installing it, I remember playing it nonstop for like at least 3 hours on young age. The beatmaps I played then are still can be found on "Most Played Beatmaps" tab on my profile page. It was so fun to play the same style concept game with variety of songs to pick on.
+
+*What made you start playing VSRG style games?*
+
+> Same as all other players, O2jam introduced me to play VSRG style games.
+>
+> Come to think of it, I was quite the VSRG enthusiast, trying various rhythm games such as O2jam, Canmusic, Audition, R2Beat, Rhythm Star, Popstage, LR2, Stepmania (Once in middle school days), DJMAX Portable/RAY, Tabsonic, etc. I played so many of them.
+>
+> I was really weak player back then, even struggling to play 3 stars (equivalent to the current osu!mania Star Rating) 7K chart.
+
+*What other rhythm games do you enjoy, besides games like O2Jam and osu!mania?*
+
+> I sometimes play Djmax respect to just ease myself. Trying new key mods, and new gimmicks.
+>
+> Also, I've been exploring Qwilight, because it's very convenient to play bms charts and also to find new gimmicks. (Also LR2 is awesome!, by the way!)
+
+*Where does the name "dressurf" come from, and is there any special meaning behind it?*
+
+> I've been anticipating this question from The Followpoint. And yes, there actually is a special meaning behind it.
+>
+> When I was using playername as anti-/anti_, I saw myself becoming better steadily at osu!mania. However, I soon realized that my name (This name is) was very common, making it challenging to stand out. So, I decided to change my name to dressurf, which is a combination of my most loved person and my favorite thing when I was a kid.
+
+*What would you say is the greatest gift that osu!mania has given you?*
+
+> Playing osu!mania gave lots of things to me, such as managing anxiety, how to schedule things between different timezones and being positive. Most importantly, playing osu!mania made me realize that nothing is impossible unless you try sincerely.
+
+*What kind of maps do you find yourself enjoying to play the most?*
+
+> I really like playing challenging technical charts, they're really fun to play and seeing the combos stack up as the notes fall relentlessly makes me amazing.
+
+*What’s your favorite score you’ve ever set, and why?*
+
+> Without a doubt, Ice angel +DT is the proudest/favorite score I ever set, because it was a turning point for me.
+>
+> After finally getting S-rank on that chart, my hand speed and reading abilities improved to new heights. I remember grinding hours and hours to just get a pass on that chart.
+
+*Did you ever see yourself becoming the rank 1 player in osu!mania, why or why not?*
+
+> Not even close, I never saw myself becoming the rank 1 when I was barely passing 6 star charts.

```

Comment:
```suggestion
> Not even close, i never saw myself becoming the #1 when i was barely passing 6* charts.

```

Reply:
"becoming the rank 1" reads better here than "becoming the #1" imo